post_installation_configuration/cluster-capabilities.adoc

```diff
@@ -6,13 +6,25 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-Cluster administrators can use the cluster capabilities to select or deselect one or more optional components prior to installation and also enable a cluster capability anytime post installation.
+Cluster administrators can use cluster capabilities to select or deselect one or more optional components prior to installation. Cluster administrators can enable disabled cluster capabilities at anytime after installation.
 
 [NOTE]
 ====
 If cluster capability has been enabled before, it cannot be disabled.
 ====
 
+include::modules/explanation-of-capabilities.adoc[leveloffset=+1]
+include::modules/cluster-bare-metal-operator.adoc[leveloffset=+2]
+include::modules/cluster-samples-operator.adoc[leveloffset=+2]
+include::modules/operator-marketplace.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../installing/installing_bare_metal_ipi/ipi-install-overview.adoc#ipi-install[Bare metal Operator]
+* xref:../operators/understanding/olm-rh-catalogs.adoc#olm-rh-catalogs[Marketplace Operator]
+* xref:../openshift_images/configuring-samples-operator.adoc#configuring-samples-operator[Samples Operator]
```

Comment:
This list is a bit brief, and it's not all that clear what to do with the entries:

![image](https://user-images.githubusercontent.com/209920/197259496-9a17e2fb-0d90-4770-88e2-361a0ba6fa75.png)

It would be nice to include these links in the earlier sections where each is discussed.  But if that is not technically possible today because of modules linking constraints, it may be worth expanding the text in these list entries to mention the slugs (`baremetal`, etc.) and to point out that sections earlier in this doc (which maybe we can link to?) go into more detail on the capability as well.  Something like:

```asciidoc
* The Bare metal Operator provides the features for the `baremetal` capability. For more information, see $THE_EARLIER_SECTION, the Cluster Operators reference, and $THE_INSTALLING_BARE_METAL_REF.
```

Reply:
Thank you for the feedback @wking! This PR is still in flight at the moment. I will reach out when I am ready for review!

Reply:
@bscott-rh Would you please make a continuous improvement issue to address @wking 's comment on this section of the docs?

If possible, I would like to keep this PR tightly scoped to the explanations of the capabilities and relationship to the Cluster Operators reference docs.